pom.xml

```
@@ -29,13 +29,15 @@
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/view-job-filters-plugin</gitHubRepo>
         <jenkins.version>2.387.1</jenkins.version>
```

Hmm,

```
Dependency io.jenkins.plugins:font-awesome-api:jar:6.4.0-2 requires Jenkins 2.387.3 or higher.
```

I think I forgot that the last active line is actually 2.401.x (though https://github.com/jenkinsci/bom/pull/2287#discussion_r1303359265), so actually
```suggestion
        <jenkins.version>2.387.3</jenkins.version>
```
after all?